### ISSUE TITLE
Issue with Slim framework

### ISSUE THREAD

**Issue by deep1931** [kingster/Way2SMS-API#8]
Hello, 
I am using your script to send sms, it is working very well with php without any framework, but when i try the same script with Slim framework it stop working.

 **Undefined property: WAY2SMSClient::$jstoken**
this is the error i am receiving. Please help, i am a mobile app developer and do not have much experience in PHP or related frameworks.

**Comment by kingster** [comment]
Could you possibly share how you are using it inside Slim? From the error message it looks some happens with the class variables inside slim.

**Comment by deep1931** [comment]
I am using in the same way as used in simple php. Previously I used your code in PHP scripts which was working very well till today, but now in the same way I have integrated with Slim but not succeed, due to lack of knowledge and time I am not able to track the issue.  
I do not have any experience in PHP or related frameworks, so with the help of google I am making web services to communicate with android app.
